### PR TITLE
Add handwired/frankie_macropad

### PR DESCRIPTION
**Pull request by eric-wood**
## Description

Hello! I've designed and built a simple macropad. You can find the PCB layouts [here](https://github.com/eric-wood/macro-pad).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/frankie_macropad/rules.mk [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE = yes
+
+EXTRAFLAGS += -flto
```
Comment (fauxpark) [inline keyboards/handwired/frankie_macropad/rules.mk:25]:
```suggestion
LTO_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/frankie_macropad/rules.mk (hunk 1 above)
Comment (eric-wood) [reply keyboards/handwired/frankie_macropad/rules.mk]: Fixed 😄

File: keyboards/handwired/frankie_macropad/readme.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# frankie_macropad
+
+![frankie_macropad](https://imgur.com/a/NqB8Fpj)
```
Comment (fauxpark) [inline keyboards/handwired/frankie_macropad/readme.md:3]:
```suggestion
![frankie_macropad](https://i.imgur.com/X46CSH7h.jpg)
```
Needs to be a direct link.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/frankie_macropad/readme.md (hunk 2 above)
Comment (eric-wood) [reply keyboards/handwired/frankie_macropad/readme.md]: Good catch! Fixed

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**